entity-framework/core/what-is-new/ef-core-10.0/whatsnew.md

```diff
@@ -21,6 +21,158 @@ EF10 requires the .NET 10 SDK to build and requires the .NET 10 runtime to run.
 
 ## Azure Cosmos DB for NoSQL
 
+<a name="full-text-search-support"></a>
+
+### Full-text search support
+
+Azure Cosmos DB now offers support for [full-text search](/azure/cosmos-db/gen-ai/full-text-search). It enables efficient and effective text searches, as well as evaluating the relevance of documents to a given search query. It can be used in combination with vector search to improve the accuracy of responses in some AI scenarios.
```

Comment:
I think the main point of FTS is about handling natural language well (e.g. stemming), may be worth mentioning that. Re the combination with vector search, I'd explicitly mention that as "hybrid" search, which is the standard term.

Reply:
I mentioned hybrid search in the section below, but makes sense to do it here also